src/Data/Connection/CommentConnectionResolver.php

```diff
@@ -253,7 +253,13 @@ public function sanitize_input_fields( array $args ) {
 	 * @return bool
 	 */
 	public function is_valid_offset( $offset ) {
-		return ! empty( get_comment( $offset ) );
+		global $wpdb;
```

Comment:
@let00 I'm going to revert this particular commit. The queries already get the cursor objects up front when doing the cursor pagination, so they'll already be cached, so calling `get_comment`, `get_post`, etc will actually be more efficient because the objects will be in memory. 

I guess I missed this commit being added before I merged.